js/modules/networkgraph/networkgraph.src.js

```diff
@@ -1056,12 +1056,4 @@ seriesType('networkgraph', 'line',
  * @product   highcharts
  * @apioption series.networkgraph.nodes.mass
  */
-/**
```

Comment:
This is result of an older merge conflict and has to be part of the TS file. Additionally the type naming is not correct instead of `...Network...` it hast to be `...Networkgraph...`. Can you fix it, please? Then I do not have to mix it with my migration PR.

Reply:
Thanks @bre1470 👍 Please see latest changes for fix.

Reply:
👍 Looks good. Thank you!